configs/tdh-test.yml

```diff
@@ -0,0 +1,296 @@
+          unit_size: 600000   # disk size / doc count of icds @ 2017-12-13
+    process:
+      cores_per_node: 4
+      ram_per_node: 8
+
+  es_datanode:
+    usage_capacity_per_node: 15000
+    min_nodes: 3
+    storage:
+      group: 'SAS'
+      redundancy_factor: 2
+      data_models:
+        - referenced_field: 'forms_total'
+          unit_size: 5500
+        - referenced_field: 'cases_total'
+          unit_size: 1800
+    process:
+      cores_per_node: 8
+      ram_per_node: 32
+
+  es_master:
+    static_number: 6
+    storage_scales_with_nodes: True
+    storage:
+      group: 'VM_other'
+      static_baseline: 100GB
+      override_storage_buffer: 0
+      override_estimation_buffer: 0
+    process:
+      cores_per_node: 2
+      ram_per_node: 8
+
+  riakcs:
+    usage_capacity_per_node: 50000
+    # avg attachment size of 12560 bytes (11000*0.96 + 50000*0.04)
+    # RAM requirement per key = 130b
+    # num keys = 10TB / (12560b x 3<redundancy factor>)
+    # RAM needed = 130b x num keys = 35GB (64GB avail per node)
+    max_storage_per_node: 25TB
+    min_nodes: 10
+    storage:
+      group: 'SSD'
+      redundancy_factor: 3
+      static_baseline: 10TB  # to account for exports etc
+      data_models:
+        - referenced_field: 'forms_total'  # 96% of objects
+          unit_size: 11000
+    process:
+      # current load quite low (~15%) (2018-10-11)
+      cores_per_node: 4
+      # need to be able to fit all keys in RAM since we're using bitcask backend
+      # Current usage is at 30% (2018-10-11)
+      ram_per_node: 4
+      ram_model:
+      - referenced_field: 'forms_total'
+        # key size (45 + 6 + 79) (overhead + bucket + key)
+        # bucket = 'blobdb'
+        # new keys are smaller but stick with old key length for safety:
+        #   new: form/xxxxxxxxxxxxxxuuidxxxxxxxxxxxxxx/Xpi-XM9CZvQ
+        #   old: form/xxxxxxxxxxxxxxuuidxxxxxxxxxxxxxx/form.xml.xxxxxxxxxxxxxxuuidxxxxxxxxxxxxxx
+        unit_size: 130
+      ram_redundancy_factor: 3
+      ram_static_baseline: 1  # per node
+
+  pg_ucr:
+    usage_capacity_per_node: 100000
+    storage_scales_with_nodes: True
+    min_nodes: 2
+    storage:
+      # This is a rough estimate.
+      # The person case UCR is 35% of total UCR usage.
+      group: 'SSD'
+      data_models:
+        - referenced_field: 'cases_total'  # cumulative
+          unit_size: 4000  # to account for monthly data etc.
+    process:
+      cores_per_node: 4
+      ram_per_node: 16
+
+  pg_warehouse:
+    usage_capacity_per_node: 300000
+    storage_scales_with_nodes: True
+    storage:
+      group: 'SSD'
+      data_models:
+        - referenced_field: 'forms_total'
+          unit_size: 4000  # to account for monthly data etc.
+    process:
+      cores_per_node: 4
+      ram_per_node: 16
+
+  pillowtop:  # will need to revamp this once combined pillows are rolled out
+    storage_scales_with_nodes: True
+    process:
+      cores_per_node: 16
+      ram_per_node: 128
```

Comment:
This should probably be 8/64 for them. They won't be running custom pillows and for some reason this estimate is coming out with two pillow servers while prod only has one of these

I thnk this should change to:


```
  pillowtop:
    static_number: 1
    storage:
      group: 'VM_other'
      static_baseline: 100GB
      override_storage_buffer: 0
      override_estimation_buffer: 0
    process:
      cores_per_node: 8
      ram_per_node: 64
```